plugins/integrations/kubernetes-service/src/main/java/com/cloud/kubernetes/cluster/actionworkers/KubernetesClusterScaleWorker.java

```diff
@@ -299,8 +298,8 @@ private void scaleKubernetesClusterOffering() throws CloudRuntimeException {
             boolean result = false;
             try {
                 result = userVmManager.upgradeVirtualMachine(userVM.getId(), serviceOffering.getId(), new HashMap<String, String>());
-            } catch (ResourceUnavailableException | ManagementServerException | ConcurrentOperationException | VirtualMachineMigrationException e) {
-                logTransitStateAndThrow(Level.ERROR, String.format("Scaling Kubernetes cluster : %s failed, unable to scale cluster VM : %s", kubernetesCluster.getName(), userVM.getDisplayName()), kubernetesCluster.getId(), KubernetesCluster.Event.OperationFailed, e);
+            } catch (CloudRuntimeException | ResourceUnavailableException | ManagementServerException | VirtualMachineMigrationException e) {
```

Comment:
is `ConcurrentOperationException` no longer an issue?

Reply:
ConcurrentOperationException is extending CloudRuntimeException, so its not required or it is not allowed to have both.

`public class ConcurrentOperationException extends CloudRuntimeException {`